extensions/ql-vscode/src/model-editor/method-modeling/method-modeling-view-provider.ts

```diff
@@ -58,6 +58,7 @@ export class MethodModelingViewProvider
 
     this.webviewView = webviewView;
 
+    this.setInitialState();
```

Comment:
Would it make sense to pass in the `webviewView` (or `webviewView.webview`) to `setInitialState` to remove the use of the `this.webviewView?` in `setInitialState`? That would make it clearer that the message will always be posted and it's not conditional.